website/README.md

```diff
@@ -1,72 +1,57 @@
-https://github.com/apache/beam-site, but is not yet ready. Please 
-continue to contribute changes at 
-[apache/beam-site](https://github.com/apache/beam-site) according to 
-the [website contribution guide](https://beam.apache.org/contribute/#contributing-to-the-website). 
-You can track migration progress via 
-[[BEAM-4493]](https://issues.apache.org/jira/browse/BEAM-4493).**_
-
-This is the website for [Apache Beam](https://beam.apache.org/), hosted at:
-
-    https://beam.apache.org/
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+These are the main sources of the website for Apache Beam, hosted at
+https://beam.apache.org/.
 
 ## About this site
 
 The Beam website is built using [Jekyll](http://jekyllrb.com/). Additionally,
 for additional formatting capabilities, this website uses
 [Twitter Bootstrap](http://getbootstrap.com/).
 
-### Repository Structure
-
-This repository contains:
-
-1. `src/`: the source of the site, including markdown files containing the bulk of the content
-1. `content/`: html generated from the markdown (which is what is actually hosted on the website)
-
-## Development Workflow
-
-### Setup
-
-You need Ruby version >= 2.2.0 to build the project.
-
-Install [Ruby Gems](https://rubygems.org/pages/download), a package management framework for Ruby.
+Documentation generated from source code, such as Javadoc and Pydoc, is stored
+separately on the [beam-site
+repository](https://github.com/apache/beam-site/tree/release-docs).
 
-Install [Bundler](http://bundler.io/v1.3/rationale.html), which  we use to specify dependencies and ensure
-a consistent environment for building the website, even across multiple developers on different machines:
+## Active development
 
-    $ gem install bundler
+Website development requires Docker installed if you wish to preview changes and
+run website tests.
 
-Use Bundler to download the versions of each dependency specified in the website's `Gemfile.lock`,
-including [Jekyll](https://jekyllrb.com/):
+The following command is used to build and serve the website locally.
 
-    $ bundle install --deployment
+    $ ./gradlew -PpublishJekyllPort :beam-website:serveWebsite
```

Comment:
Is specifying `-PpublishJekyllPort` necessary? It would be nice to make the default invocation as simple as possible.

Reply:
I think it's simple: you just copy and paste the line.

Reply:
I don't know of a better way to do this.